test/mocks/extended_playbook_config/playbooks/unscheduled_playbook/playbook.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
it looks like this mock is the same as the `scheduledplaybook` https://github.com/twilio-labs/sls-apb/pull/23/files#diff-aa65a8ae6bcb77d98b7e7f05cbe37dea42c02a7dcb39f354092f7bcefb2cd870

which also don't contain any code specific to the playbooks themselves, can/should we use one of the existing general playbook mocks instead?

Reply:
Yeah I'll take a look at how we can reuse.

Reply:
Will circle back to this one later. I'm not seeing the obvious way to implement it right now without refactoring the existing playbook mocks since these specific files need to be json